openerp/models.py

```diff
@@ -638,10 +554,26 @@ def _build_model(cls, pool, cr):
         # determine the attributes of the model's class
         ModelClass._build_model_attributes(pool)
 
-        # instantiate the model, and initialize it
+        check_pg_name(ModelClass._table)
+
+        # Transience
+        if ModelClass._transient:
+            ModelClass._transient_check_count = 0
+            ModelClass._transient_max_count = config.get('osv_memory_count_limit')
```

Comment:
Well, it was written in a way that makes overriding cumbersome. At least the override you point does not work, since this one overrides the value of `_transient_max_hours` from the class :frowning: